app/models/message.rb

```diff
@@ -39,8 +38,4 @@ def validate_conversation_user
   def start_assistant_reply
     conversation.messages.create! role: :assistant, content_text: "", assistant: conversation.assistant
   end
-
-  def broadcast_message
-    broadcast_append_to conversation, partial: "messages/message", locals: { scroll_down: true, timestamp: (Time.current.to_f*1000).to_i }
-  end
```

Comment:
@robacarp Notably, I was still seeing some flickering occasionally at the beginning of message generation. I finally tracked it down to this.

After you submit the form with a new chat message, these initial messages appear in the body with the full page render / morphing. However, we were also broadcasting them and so there was a race condition where sometimes the broadcast would happen before the morph had completed so when the morph ran it would roll back a moment in time. So we actually only need the broadcasting for updating the assistant's reply (within the job).